src/main/drivers/accgyro/accgyro_spi_lsm6dso_init.c

```diff
@@ -40,6 +40,9 @@
 
 #define LSM6DSO_CHIP_ID 0x6C
 
+// equivalent to 70 mdps/LSB, as specified in LSM6DSO datasheet section 4.1, symbol G_So
+#define LSM6DSO_GYRO_SCALE_2000DPS 14.285714286f
```

Comment:
This was `#define GYRO_SCALE_2000DPS (2000.0f / (1 << 15))` or `2000.0/0x8000` = `0.061`.

I believe you should have `#define LSM6DSO_GYRO_SCALE_2000DPS 0.070f`

Reply:
I agree, I think I was confused by the comment for GYRO_SCALE_2000DPS. For some reason the comment contains the lsb/DPS value, but the value of the variable is in DPS/lsb

Reply:
Should be fixed now